Revert Android build error introduced in version 2.0.76

Revert the changes in https://github.com/elimu-ai/model/pull/305

https://github.com/elimu-ai/appstore/actions/runs/13391715387/job/37400718024

```
> Task :app:mergeExtDexDebug
ERROR:/home/runner/.gradle/caches/transforms-3/c4785769149225e742e98f5eef5b76e2/transformed/jetified-error_prone_annotations-2.36.0.jar: D8: java.lang.NullPointerException

> Task :app:mergeExtDexDebug FAILED
> Task :app:processDebugManifestForPackage

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:mergeExtDexDebug'.
> Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
   > Failed to transform error_prone_annotations-2.36.0.jar (com.google.errorprone:error_prone_annotations:2.36.0) to match attributes {artifactType=android-dex, asm-transformed-variant=NONE, dexing-enable-desugaring=true, dexing-is-debuggable=true, dexing-min-sdk=24, org.gradle.category=library, org.gradle.libraryelements=jar, org.gradle.status=release, org.gradle.usage=java-runtime}.

      > Execution failed for DexingNoClasspathTransform: /home/runner/.gradle/caches/transforms-3/c47857[69](https://github.com/elimu-ai/appstore/actions/runs/13391715387/job/37400718024#step:5:70)149225e742e98f5eef5b76e2/transformed/jetified-error_prone_annotations-2.36.0.jar.
Deprecated Gradle features were used in this build, making it incompatible with Gradle 8.0.
         > Error while dexing.
```

--- a/pom.xml
+++ b/pom.xml
@@ -86,7 +86,7 @@
     <dependency>
       <groupId>com.google.code.gson</groupId>
       <artifactId>gson</artifactId>
-      <version>2.12.1</version>
+      <version>2.11.0</version>
     </dependency>
     <!-- Lombok -->
     <dependency>

